olive/workflows/run/config.py

```diff
@@ -192,12 +190,7 @@ def _resolve_evaluator(v, values):
         return v
 
     evaluator = v.get("evaluator")
-    if isinstance(evaluator, dict):
-        v["evaluator"] = _resolve_system(evaluator, values, "target")
-        for metrics in v["evaluator"].get("metrics", []):
```

Comment:
LIne 195 is not used but I think 197-... is required. The evaluator for passes can be specified as dict. They might have `data_config` that need to be resolved.

Reply:
Maybe it can be changed to be similar to line 110 under `resolve_evaluators` 
```
if isinstance(evaluator_dict): 
   for idx, metric in enumerate(v["evaluator].get("metrics", [])):
            v["evaluator"]["metrics"][idx] = _resolve_data_config(metric, values, "data_config")
``` 

This way it is clear that the original `v` is being updated.